guides/rails_guides/markdown.rb

```diff
@@ -82,6 +84,11 @@ def generate_header
         @header = engine.render(@raw_header).html_safe
       end
 
+      def generate_description
+        sanitizer = Rails::Html::FullSanitizer.new
+        @description = sanitizer.sanitize(@header).gsub(/\R/, " ")
```

Comment:
Instead of `gsub(/\R/, " ")` you can use [`.squish`](https://api.rubyonrails.org/classes/String.html#method-i-squish) which should give the similar result. 

```suggestion
        @description = sanitizer.sanitize(@header).squish
```

Reply:
Good catch! thanks 😸